app_specific_files/dummy_app_multicast/scripts/generate_random_files.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+: '
+    ** Copyright (c) 2019, Autonomous Networks Research Group. All rights reserved.
+    **     contributor: Quynh Nguyen, Bhaskar Krishnamachari
+    **     Read license file in main directory for more details
+'
+
+file_path=$1
+file_size=$2
+echo 'Create file '$file_name
+echo 'File size '$file_size
+dd if=/dev/urandom of=$file_path bs=1K count=$file_size #KBytes
```

Comment:
Need newline

Reply:
newline? I think when the logs are generated, they are separate lines already? This is bash script.

Reply:
Because of historical reasons, many compilers expect a newline at the end of every file. Because of this, it's just an accepted standard for any and all files for any language.